### DISCUSSION TITLE
Can't play two videos simultaneously on Android?

### DISCUSSION THREAD

**Discussion by Jason0247** [airsdk/Adobe-Runtime-Support#2806] [Q&A]
The AIR SDK I am using is 50.2.2.5, and when I try to display two Video objects at the same time, there will always be one that is not visible.
Is it under the Android system that AIR does not allow two videos to be displayed simultaneously?
Thank you!

**Comment by ajwfrost** [comment]
Yes, on Android the hardware decoding for H.264 video is limited to one at a time. This restriction was added right at the start by Adobe though, I'm not actually sure whether it should be relevant for today's devices .. we can investigate a little to see whether it could be adjusted now.

thanks

**Reply by itlancer** [reply]
@ajwfrost, yes, please. Multiple videos playback on Android essential for us too.

**Reply by Jason0247** [reply]
Excuse me, is there a specific adjustment time for this issue?
thanks

**Comment by Jason0247** [comment]
@ajwfrost Displaying multiple videos (at least two) on Android is indeed important. thanks
